### PR TITLE
Changing Edge Startup URL post

### PR DESCRIPTION
**Pull request by doctordns**
# PR Summary

This is another community blog post article on how to change the Edge startup page URL(s).
It is based on an earlier Scripting Guys post o how to change IE's home page. 

## PR Checklist

- Do you want this post to be published on a specific date?: Y/N [Specify date]:
- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] Includes content related an open issue
  - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: Posts/2021/06/tfl-edgestart.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+**Q:** How can I change the Edge startup page?
+
+**A:** You can change the start page easily using PowerShell.
+
+## Edge and It's Start Page
+
+I am basing this article on the latest incarnation of the Edge browser, aka Edge Chromium.
+The settings in this article seem to work fine on the latest versions of Windows 10 and Server 2022.
+Other browsers can have different approaches to startup page.
+And as ever - E&OE!
```
Comment (sdwheeler) [inline Posts/2021/06/tfl-edgestart.md:17]: what does "E&OE" mean?

**Review by sdwheeler — CHANGES_REQUESTED** [review]
A few minor changes

File: Posts/2021/06/tfl-edgestart.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+For many, this is fine - they can browse the contents and then navigate where they want.
+But in some circumstances, you may wish to change this default.
+And fortunately, this is straightforward to achieve.
+
+An easy way to set the startup page for yourself is to configure two registry value entries on the local machine.
+The first is the **RestoreOnStartup** value entry to the registry key **HKCU:\\Software\\Policies\\Microsoft\\Edge**.
+This value entry is a **REG_DWORD**.
+By setting this entry with a value of **4**, you tell Edge to use the URL or URLs you specify when it starts up rather than the default home page.
+
+The second value entry (or entries) is within the key **HKCU:\\Software\\Policies\\Microsoft\\Edge\\RestoreOnStartupURLs**.
```
[the remark this answers is not in the document]
Comment (doctordns) [reply Posts/2021/06/tfl-edgestart.md:29]: ok

File: Posts/2021/06/tfl-edgestart.md (hunk 1 above)
Comment (doctordns) [reply Posts/2021/06/tfl-edgestart.md:17]: Errors and Omissions Excepted.

**Comment by doctordns** [comment]
Ready to go.

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**